test/integration/lighthouse.js

```diff
@@ -1 +1,38 @@
-require('lighthouse');
+const chrome = require('chrome-aws-lambda');
+const { launch } = require('puppeteer-core');
```

Comment:
As described by @lucleray I think because this is a class method, you want to make sure you do `const pupeteer = ` and then call `pupeteer.launch`.

Reply:
```suggestion
const puppeteer = require('puppeteer-core');
```